rest/plugin.spec.yaml

```
@@ -71,6 +71,13 @@ connection:
     type: credential_username_password
     required: false
     example: '{"username": "user@example.com", "password": "mypassword"}'
+  fail_on_error:
```

I think we can make this more specific

```suggestion
  fail_on_http_errors:
```